contracts/HATVaults.sol

```diff
@@ -1138,26 +1133,13 @@ contract  HATVaults is Governable, ReentrancyGuard {
         uint256 totalSupply = poolInfos[_pid].balance;
         require(totalSupply > 0, "HVE28");
         require(_severity < bountyInfos[_pid].bountyLevels.length, "HVE06");
-        uint256 totalBountyAmount =
-        totalSupply * bountyInfos[_pid].bountyLevels[_severity];
-        claimBounty.hackerVested =
-        totalBountyAmount * bountyInfos[_pid].bountySplit.hackerVested
-        / (HUNDRED_PERCENT * HUNDRED_PERCENT);
-        claimBounty.hacker =
-        totalBountyAmount * bountyInfos[_pid].bountySplit.hacker
-        / (HUNDRED_PERCENT * HUNDRED_PERCENT);
-        claimBounty.committee =
-        totalBountyAmount * bountyInfos[_pid].bountySplit.committee
-        / (HUNDRED_PERCENT * HUNDRED_PERCENT);
-        claimBounty.swapAndBurn =
-        totalBountyAmount * bountyInfos[_pid].bountySplit.swapAndBurn
-        / (HUNDRED_PERCENT * HUNDRED_PERCENT);
-        claimBounty.governanceHat =
-        totalBountyAmount * bountyInfos[_pid].bountySplit.governanceHat
-        / (HUNDRED_PERCENT * HUNDRED_PERCENT);
-        claimBounty.hackerHat =
-        totalBountyAmount * bountyInfos[_pid].bountySplit.hackerHat
-        / (HUNDRED_PERCENT * HUNDRED_PERCENT);
+        uint256 totalBountyAmount = totalSupply * bountyInfos[_pid].bountyLevels[_severity] / HUNDRED_PERCENT;
```

Comment:
this change actually changes the rounding behavior, so now things are less precise. Perhaps not significant, but still better not to change this